src/Pos/Wallet/Web/Tracking.hs

```diff
@@ -128,7 +129,7 @@ syncWalletsAtStartWebWallet :: WalletTrackingEnv ext m => [EncryptedSecretKey] -
 syncWalletsAtStartWebWallet = syncWalletsWithGStateLock @WalletSscType
 
 syncOnImportWebWallet :: WalletTrackingEnv ext m => EncryptedSecretKey -> m ()
-syncOnImportWebWallet = selectAccountsFromUtxoLock @WalletSscType . one
+syncOnImportWebWallet = (() <$) . selectAccountsFromUtxoLock @WalletSscType . one
```

Comment:
How come that this change (and quite a few of other changes) is displayed in this PR (#1045)? It has no relation to `Arbitrary` and belongs to a different branch. You should've forked from a recent `master`.